common_test.go

```diff
@@ -0,0 +1,35 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tools
+
+import (
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestFindRepoRoot(t *testing.T) {
+	// prepare
+	expected, _ := filepath.Abs("./")
+
+	// test
+	actual, err := FindRepoRoot()
+
+	// verify
```

Comment:
nit: these comments are pretty superficial. The variable names and context of the function convey the same information.

```suggestion
	expected, _ := filepath.Abs("./")
	actual, err := FindRepoRoot()
```